pkg/mtbroker/filter/filter_handler.go

```diff
@@ -217,17 +217,8 @@ func (h *Handler) send(ctx context.Context, writer http.ResponseWriter, headers
 	statusCode, err := writeResponse(ctx, writer, response, ttl, span)
```

Comment:
You might need to remove the span here or adjust the signature below for tests to pass?